app/src/main/java/net/squanchy/navigation/view/NoSwipeViewPager.java

```diff
@@ -0,0 +1,23 @@
+package net.squanchy.navigation.view;
+
+import android.content.Context;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+public class NoSwipeViewPager extends ViewPager {
```

Comment:
❌  this is not preventing keyboard navigation, you need to override another method. Off the top of my head:

```java
@Override
public boolean executeKeyEvent(KeyEvent e) {
  return false;
}
```